students/scotchwsplenda/Lesson_3/MailRoom.py

```diff
@@ -0,0 +1,111 @@
+import sys  # need to do this if you want to use sys.exit()
+from operator import itemgetter
+
+
+# donors
+donators = [
+("Gordian",[30.0,45.0]),
+("Tiberius",[60.0]),
+("Maximus",[65.0, 12.0]),
+("Tacitus",[33.0,22.0,25.00]),
+("Commodus",[43.0,11.0])]
+
+# Names of donors
+extant_don=[i[0] for i in donators]
```

Comment:
I wouldn't generate the names separately - you would then need to maintain the names list separately.